lb_pid: {'kp': 25, 'ki': 18, 'kd': 18, 'ir': 75}#{'kp': 22, 'ki': 12, 'kd': 13, 'ir': 74}#{'kp': 25, 'ki': 20, 'kd': 20, 'ir': 75}
lb_serial_number: 'FT3X3ISJ'
lb_max_speed: 2000

lf_pid: {'kp': 30, 'ki': 23, 'kd': 18, 'ir': 75}
lf_serial_number: 'FT4IT45E'
lf_max_speed: 2000


rb_pid: {'kp': 30, 'ki': 21, 'kd': 18, 'ir': 75}
rb_serial_number: 'FT4ITA2X'
rb_max_speed: 2000

 
rf_pid: {'kp': 25, 'ki': 18, 'kd': 18, 'ir': 75}#{'kp': 33, 'ki': 13, 'kd': 18, 'ir': 75}
rf_serial_number: 'FT4JQ6DY'
rf_max_speed: 2000

angle_tolerance: 1.5
